hack/package.sh

```diff
@@ -96,6 +139,9 @@ function package() {
   # shellcheck disable=SC2155
   local context="$(setup_image_package_context "${target}" "${task}" "${path}")"
 
+  setup_release "${context}"
+  setup_tencent_cos "${context}"
```

Comment:
Do we need to check `PACKAGE_BUILD=false` before setup release?

Reply:
that's ok, we just do these steps before PACKAGE_BUILD.